Source/WebCore/platform/ContentType.h

```diff
@@ -28,12 +28,18 @@
 
 #include <wtf/text/WTFString.h>
 
+namespace WTF {
+class URL;
+}
+
 namespace WebCore {
 
 class ContentType {
 public:
     WEBCORE_EXPORT explicit ContentType(String&& type);
     WEBCORE_EXPORT explicit ContentType(const String& type);
+    WEBCORE_EXPORT ContentType(const String& type, const WTF::URL&);
+    WEBCORE_EXPORT ContentType(const String& type, bool);
```

Comment:
Is this second constructor really needed? I don't see it used anywhere.

Reply:
it's used by the IPC serialiser. It is very weird how the serialiser works IMHO. I'll add a comment

Reply:
You could make it private and mark the serializer class as friend like we often do.

Reply:
I now use it following the move to have a `fromURL()` method